### DISCUSSION TITLE
Can not run testbed fox demo [CUDA_ERROR_OUT_OF_MEMORY]

### DISCUSSION THREAD

**Discussion by Zekiah-A** [NVlabs/instant-ngp#401] [Q&A]
When attempting to run the fox testbed demo, the window opens for a short amount of time, and instantly closes, giving me a CUDA_ERROR_OUT_OF_MEMOR, I was unable to find any fixes of this in my freetime, and any help would be appreciated.

❯ ./build/testbed --scene data/nerf/fox
18:27:11 INFO     Loading NeRF dataset from
18:27:11 INFO       data/nerf/fox/transforms.json
18:27:11 SUCCESS  Loaded 50 images of size 1080x1920 after 0s
18:27:11 INFO       cam_aabb=[min=[1.0229,-1.33309,-0.378748], max=[2.46175,1.00721,1.41295]]
18:27:12 INFO     Loading network config from: configs/nerf/base.json
18:27:12 INFO     GridEncoding:  Nmin=16 b=1.51572 F=2 T=2^19 L=16
Warning: FullyFusedMLP is not supported for the selected architecture 61. Falling back to CutlassMLP. For maximum performance, raise the target GPU architecture to 75+.
Warning: FullyFusedMLP is not supported for the selected architecture 61. Falling back to CutlassMLP. For maximum performance, raise the target GPU architecture to 75+.
Warning: FullyFusedMLP is not supported for the selected architecture 61. Falling back to CutlassMLP. For maximum performance, raise the target GPU architecture to 75+.
18:27:12 INFO     Density model: 3--[HashGrid]-->32--[FullyFusedMLP(neurons=64,layers=3)]-->1
18:27:12 INFO     Color model:   3--[Composite]-->16+16--[FullyFusedMLP(neurons=64,layers=4)]-->3
18:27:12 INFO       total_encoding_params=13074912 total_network_params=9728
18:27:12 ERROR    Uncaught exception: /home/zekiah/instant-ngp/dependencies/tiny-cuda-nn/include/tiny-cuda-nn/gpu_memory.h:550 cuMemCreate(&m_handles.back(), n_bytes_to_allocate, &prop, 0) failed with error CUDA_ERROR_OUT_OF_MEMORY
Could not free memory: /home/zekiah/instant-ngp/dependencies/tiny-cuda-nn/include/tiny-cuda-nn/gpu_memory.h:459 cuMemRelease(handle) failed with error CUDA_ERROR_INVALID_VALUE

I am on:
```
OS: Arch Linux x86_64
CPU: Intel i5-7300HQ
GPU0: Intel HD Graphics 630
GPU1: NVIDIA GeForce GTX 1050 Mobile
RAM: 8GB
NVIDIA DRIVER: 510.54
```

**Comment by Tom94** [comment]
Hi there, your graphics card has too little VRAM (2 GB) to run the `fox` example. 2 GB is quite little, so I don't see an easy way to squeeze even a smaller dataset in there. Sorry!
